Source/embeddings/CouldNotResolveDeletionToEvents.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+export class CouldNotResolveDeletionToEvents extends Error {
+    /**
+     * Initializes a new instance of {@link CouldNotResolveDeletionToEvents}
+     */
+    constructor() {
+        super('resolveDeletionToEvents has failed to process');
```

Comment:
```suggestion
        super('Embeddings deletion method didn't return any events. Make sure that the method marked with the @resolveDeletionToEvents decorator or the inline builders resolveDeletionToEvents() methods return one or many events.');
```

What do you think of this renaming? Super verbose, but it would be specific here.